WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -301,6 +288,22 @@ + (void)configureDefaultKeyboardPreferences
   dlclose(handle);
 }
 
++ (void)forceTurnOnSoftwareKeyboard
```

Comment:
I think it might be a good idea to add `Simulator` to the method name to avoid unnecessary confusion